docs/staking-entry-queue/notebook/sample.py

```diff
@@ -0,0 +1,161 @@
+import marimo
+
+__generated_with = "0.13.15"
+app = marimo.App(width="medium")
+
+
+@app.cell
+def _():
+    import marimo as mo
+    import math
+    import matplotlib.pyplot as plt
+    import numpy as np
+    from scipy.stats import hypergeom
+    return hypergeom, math, mo, plt
+
+
+@app.cell(hide_code=True)
+def _(mo):
+    existing_validators = mo.ui.slider(
```

Comment:
You can do something like `show_value=True` which is quite nice for seeing the value without needing to hover. If you make the full width as well also simpler to hit certain values.

Alternative approach:

```python
existing_validators = mo.ui.slider(
    100,
    8192,
    value=4096,
    label="Total Existing Validators",
    show_value=True,
    full_width=True,
)
new_validators_added = mo.ui.slider(
    1,
    256,
    value=204,
    label="New Validators Added to Pool",
    show_value=True,
    full_width=True,
)
committee_size = mo.ui.slider(
    32, 64, value=48, label="Committee Size", show_value=True, full_width=True
)

mo.vstack([existing_validators, new_validators_added, committee_size])
```